thrust/detail/type_traits.h

```diff
@@ -144,11 +144,11 @@ template<typename T> struct has_trivial_constructor
       is_pod<T>::value
 #if THRUST_HOST_COMPILER == THRUST_HOST_COMPILER_MSVC || \
     THRUST_HOST_COMPILER == THRUST_HOST_COMPILER_CLANG
-      || __has_trivial_constructor(T)
+      || __is_trivially_copyable(T)
```

Comment:
But this should be `__is_trivially_constructible` right? Its listed on the same clang page.